android/src/main/java/com/revenuecat/purchases/hybridcommon/mappers/StoreProductMapper.kt

```diff
@@ -1,11 +1,36 @@
 package com.revenuecat.purchases.hybridcommon.mappers
 
 import com.revenuecat.purchases.ProductType
+import com.revenuecat.purchases.models.PricingPhase
 import com.revenuecat.purchases.models.StoreProduct
 
+// TODOBC5: JOSH's temporary (???) extensions to support what was already there
+val StoreProduct.priceAmountMicros: Long
+    get() = this.oneTimeProductPrice?.priceAmountMicros ?: this.defaultOption?.pricingPhases?.lastOrNull()?.priceAmountMicros ?: 0
+val StoreProduct.price: String
+    get() = this.oneTimeProductPrice?.formattedPrice ?: this.defaultOption?.pricingPhases?.lastOrNull()?.formattedPrice ?: ""
```

Comment:
all of this could be cleaner if we put base plan fields into the StoreProduct... juuuuust saying ;)